optuna/integration/tensorflow.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import absolute_import
+
+import optuna
+
+
+try:
+    import tensorflow as tf
+    from tensorflow.train import SessionRunHook
+    _available = True
+except ImportError as e:
+    _import_error = e
+    # PruningHook is disabled because TensorFlow is not available.
```

Comment:
I think this `PruningHook` means `TensorFlowPruningHook`. Could you use the full name of hook as`TensorFlowPruningHook` like other integration modules?

Reply:
I fixed it as you proposed.